### ISSUE TITLE
No pinned source message showing up with pinned sources 

### ISSUE THREAD

**Issue by mobi2002** [tachiyomiorg/tachiyomi#8919]
### Steps to reproduce

Enable "only search in pinned sources"
Pin at least one source
Tap on global search 

### Expected behavior

No pinned source warning not shown 

### Actual behavior

No pinned source warning shown

### Tachiyomi version

r5452

### Android version

Android 13

### Device

Tecno spark 6

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
Yes I'll leave this open until the message is added back properly.
